modules/@apostrophecms/doc/index.js

```diff
@@ -1288,6 +1289,19 @@ module.exports = {
       addCacheFieldMigration() {
         self.apos.migration.add('add-cache-invalidated-at-field', self.setCacheField);
       },
+
+      async addSetPreviousDocsAposModeMigration () {
+        self.apos.migration.add('set-previous-docs-apos-mode', async () => {
+          await self.apos.doc.db.updateMany({
+            _id: { $regex: ':previous' }
```

Comment:
Good, but speed this up by also checking that `aposMode` is *not* `previous`. Reduces the number of docs that have to be touched by mongodb.

Also, just to be thorough please root that regex with `$` at the end.